### DISCUSSION TITLE
Migration Error

### DISCUSSION THREAD

**Discussion by pce-owa** [atrocore/atrocore#1137] [Q&A] [answered]
Hello, 

i need help please. 

I try to update core from 1.8.5 but it breaks here (migration to 1.8.6):

![image](https://github.com/atrocore/atrocore/assets/147825555/a8db9372-7bb5-4b8f-b9c3-601c56cef846)

Current Versions:
![image](https://github.com/atrocore/atrocore/assets/147825555/f9296c6e-a822-46b3-8e6e-0a4453f71c51)

Thank you very much

**Comment by rratsun** [comment]
Sorry for delay. I have created an internal issue for it. Our developer try to understand the reasons. We will write you soon.

P.S. I see from the screenshot that you have a premium module which license has expired. This may be the reason because this module can no longer be updated and it somehow conflicts with the migration of the Core. I recommend you to renew your license.
